uber/templates/emails/reg_workflow/attendee_confirmation.html

```diff
@@ -14,6 +22,11 @@
 {% endif %}
 extra money <a href="{{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}">here</a>.
 
+{% if attendee.paid == c.NEED_NOT_PAY and not attendee.promo_code_id or attendee.is_group_leader %}
```

Comment:
While there's any group member in a free group could have their badge filled out by an admin, I figured it was slim enough chance that we don't need to show this disclaimer to everyone in a free group. I might change my mind on that, though.